rholang/src/main/scala/coop/rchain/rholang/interpreter/registry/Registry.scala

```diff
@@ -1,4 +1,4 @@
-package coop.rchain.rholang.interpreter
```

Comment:
The commit `Move Registry.scala` is not needed, as this PR deletes Registry.scala

Reply:
It deletes everything in Registry.scala except `buildURI` and a helper object for `buildURI`.

Reply:
:+1: Sorry, I've missed that :)